src/pages/EmployeeProfile/components/BannerItem.jsx

```diff
@@ -0,0 +1,24 @@
+//import
+import { Avatar, Paper, Stack } from "@mui/material";
+import "./style.css"
+
+import Image from '../media/plant-banner.jpg'
+
+export default function BannerItem() {
```

Comment:
Please remove the corner of the Banner, because you have a white corner
<img width="138" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/37250473/69862ccd-99ae-4e09-9f48-c8aee913d35d">